### PR TITLE
Add section about Beta releases in the README.md

### PR DESCRIPTION
**Pull request by amadeo-workos**
## Description
Add section about Beta releases in the README.md

CC @PaulAsjes 

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,19 @@ workos.client_id = "client_1234"
 
 For our SDKs WorkOS follows a Semantic Versioning ([SemVer](https://semver.org/)) process where all releases will have a version X.Y.Z (like 1.0.0) pattern wherein Z would be a bug fix (e.g., 1.0.1), Y would be a minor release (1.1.0) and X would be a major release (2.0.0). We permit any breaking changes to only be released in major versions and strongly recommend reading changelogs before making any major version upgrades.
 
+## Beta Releases
+
+WorkOS has features in Beta that can be accessed via Beta releases. We would love for you to try these
+and share feedback with us before these features reach the stable phase. To install a Beta version, please follow the
```
Comment (ameesha) [inline README.md:44]: is there a more generic terminology for "stable phase"? like "general audience (GA)" or something like that?

File: README.md [hunk 2]
```diff
@@ -38,6 +38,19 @@ workos.client_id = "client_1234"
...
+Installation steps above using the Beta release version.
```
[the remark this answers is not in the document]
Comment (ameesha) [reply README.md:45]: also can you link to the steps

[the remark this answers is not in the document]
Comment (amadeo-workos) [reply README.md]: Good idea! Just pushed a commit to add that link. Let me know what you think!

File: README.md [hunk 3]
```diff
@@ -38,6 +38,19 @@ workos.client_id = "client_1234"
 
 For our SDKs WorkOS follows a Semantic Versioning ([SemVer](https://semver.org/)) process where all releases will have a version X.Y.Z (like 1.0.0) pattern wherein Z would be a bug fix (e.g., 1.0.1), Y would be a minor release (1.1.0) and X would be a major release (2.0.0). We permit any breaking changes to only be released in major versions and strongly recommend reading changelogs before making any major version upgrades.
 
+## Beta Releases
+
+WorkOS has features in Beta that can be accessed via Beta releases. We would love for you to try these
+and share feedback with us before these features reach general audience (GA). To install a Beta version,
```
Comment (ameesha) [inline README.md:44]: general availability* (sorry i think i typo'ed my previous comment)

**Review by ameesha — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (amadeo-workos) [reply README.md]: ouch, yes! good call 😄 Thanks @ameesha !

**Review by PaulAsjes — APPROVED** [review]
Looking good! Can we get the changes that @ameesha highlighted into the other SDK readmes as well?

**Pull request merged**